src/components/training/TrainingPlanOverview.jsx

```
@@ -0,0 +1,1114 @@
+                      </div>
+                    </div>
+
+                    {/* Expanded Exercises - only show if not rest day */}
+                    {isExpanded && !isRestDay && (
+                      <div className="border-t border-zinc-700/50 bg-zinc-800/30">
+                        <div className="p-6 space-y-4">
+                          {/* Warmup section */}
+                          <div className="mb-6">
+                            <label className="block text-zinc-300 font-semibold mb-2 text-lg">
+                              Warmup
+                            </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)